gunicorn/sock.py

```
@@ -38,6 +38,11 @@ def __getattr__(self, name):
 
     def set_options(self, sock, bound=False):
         sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        try:
+            sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
+        except socket.error as ex:
+            if ex.errno == 92:
```

Also, perhaps we should raise the exception if `exc.errno` is not `92`.